lib/active_support/cache/memcached_snappy_store.rb

```diff
@@ -12,6 +12,16 @@ module Cache
     class MemcachedSnappyStore < MemcachedStore
       class UnsupportedOperation < StandardError; end
 
+      module SnappyCompressor
+        def self.compress(source)
+          Snappy.deflate(source)
+        end
+
+        def self.decompress(source)
+          Snappy.inflate(source)
+        end
+      end
```

Comment:
WDYT of keeping the interface to `#inflate` & `#deflate` since Zlib also follow the same interface?

Reply:
I had considered that.  The main advantage of this interface is that it is compatible with being re-used with Dalli using its `:compressor` option.  Additionally, this is the interface that `ActiveSupport::Gzip` implements.

Reply:
Ah TIL, sounds good to me !